lib/faye-redis-sharded.js

```diff
@@ -1,6 +1,8 @@
 var redis = require('redis'),
   ShardManager = require('./sharding/shard-manager');
 
+const sentry = require('../../../src/util/sentry');
```

Comment:
Want to log a Sentry too so we know when clientId is undefined, so include `sentry.js` from aha-faye-app and use it later on.